app/main.py

```
@@ -1,3 +1,60 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __add__(self, other):
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(self.km + other)
+
+    def __iadd__(self, other):
+        if isinstance(other, Distance):
+            self.km += other.km
+        elif isinstance(other, (int, float)):
+            self.km += other
+        return self
+
+    def __mul__(self, other):
+        if isinstance(other, (int, float)):
+            return Distance(self.km * other)
+
+    def __truediv__(self, other):
+        if isinstance(other, (int, float)):
+            return Distance(round((self.km / other), 2))
```

Ensure to handle division by zero. You can raise an exception or return a special value to indicate an error.